rfc.md

```diff
@@ -0,0 +1,58 @@
+# Friends Toy
+
+## Purpose
+This is a simple toy app designed to increase my familiarity with scala. 
+The basic purpose is to model friendships between people. 
+
+## Description
+For now all people will be held in a single "table" and will be self-joined in order to model relationships.
+We are also avoiding using an actual database (for now), choosing to focus initially on increasing familiarity with scala.
+The only initial piece of information we are holding onto about our "people" and "friends" is their names. For now we assume that names are unique.
+Upon creation, each person will be assigned a UUID that will be used to identify them.
+
+## Endpoints
+
+All `500` responses will return with `{error: 'Internal Error'}`
+
+- GET /people -> Returns a list of people
```

Comment:
Let's just use POST only and treat the service as what it truly would become, a series of RPCs. Then all requests should take in requests via the post body, not in the URI. For the same reason, let's not have you focus on response codes; simply make everything a 500 or a 200, which what happened encoded in the response. 500s should ONLY be used for internal, unrecoverable errors.